notes/ebt-benefits-stolen.vbs

```diff
@@ -0,0 +1,514 @@
+    Text 10, 135, 125, 10, "Illegal method used to steal benefits:"
+    GroupBox 5, 155, 375, 80, "Replacement of Stolen EBT Benefits (DHS-8557)"
+    Text 10, 170, 115, 10, "Date DHS-8557 mailed to the client:"
+    Text 10, 185, 120, 10, "Date DHS-8557 signed by the client:"
+    Text 10, 200, 115, 10, "Date client returned DHS-8557:"
+    Text 10, 215, 115, 10, "Action Taken on DHS-8557:"
+    GroupBox 5, 240, 375, 130, "Fraud Prevention Investigation (FPI) Referral (DHS-3335A)"
+    Text 10, 255, 145, 10, "Date DHS-3335A sent to Fraud Investigator:"
+    Text 10, 270, 90, 10, "Results of the DHS-3335A:"
+    Text 10, 285, 140, 10, "How stolen benefit replacement validated:"
+    Text 10, 300, 330, 10, "Has client requested and received previous benefits replacement between 10/1/22 through present?"
+    Text 10, 330, 295, 20, "If yes, how many other replacements due to stolen EBT benefits has this client received? (Indicate # and and benefit type replaced)"
+    Text 10, 385, 60, 10, "Worker Signature: "
+  EndDialog
+
+  'Validation for dialog fields
+  Do
+    Do
+      err_msg = ""
+      DIALOG dialog1
+      Cancel_confirmation
+      Call validate_MAXIS_case_number(err_msg, "*")
+      If request_status = "Select one:" Then err_msg = err_msg & vbCr & "* You must select an option for 'Decision on Request'."
+      
+      'Error handling for EBT Stolen Benefit Details groupbox
+      If IsDate(trim(date_client_reported)) <> True or len(trim(date_client_reported)) > 10 or len(trim(date_client_reported)) < 6 Then err_msg = err_msg & vbCr & "* You must enter the date the client reported stolen benefit in the format MM/DD/YYYY."
+      If trim(stolen_benefit_type) = "" Then err_msg = err_msg & vbCr & "* You must enter the type of benefit stolen."
+      If IsDate(trim(date_client_discovered_stolen_benefit)) <> True or len(trim(date_client_discovered_stolen_benefit)) > 10 or len(trim(date_client_discovered_stolen_benefit)) < 6 Then err_msg = err_msg & vbCr & "* You must enter the date the client discovered the stolen benefit in the format MM/DD/YYYY."
+      If trim(dollar_value_stolen_benefit) = "" Then err_msg = err_msg & vbCr & "* You must enter the dollar value of the stolen benefit."
+      If trim(illegal_method_used) = "" Then err_msg = err_msg & vbCr & "* You must enter the illegal method used to steal benefits."
+      
+      'Validation for DHS-8557 details groupbox
+      If trim(date_dhs_8557_sent) <> "" and IsDate(date_dhs_8557_sent) <> True or len(trim(date_dhs_8557_sent)) > 10 or len(trim(date_dhs_8557_sent)) < 6 Then err_msg = err_msg & vbCr & "* You must enter the date the DHS-8557 was mailed to the client in the format MM/DD/YYYY."
+      If trim(date_dhs_8557_signed) <> "" and IsDate(date_dhs_8557_signed) <> True or len(trim(date_dhs_8557_signed)) > 10 or len(trim(date_dhs_8557_signed)) < 6 Then err_msg = err_msg & vbCr & "* You must enter the date the client signed the DHS-8557 in the format MM/DD/YYYY."
```

Comment:
Need more conditionals on the error handling - it requires things like date the resident signed 8557, even if selecting "Awaiting return of DHS-8557". Add handling throughout this section so that mandatory fields align with dropdown choices for case status.